packages/orchestration/test/exos/local-orchestration-account-kit.test.ts

```diff
@@ -284,3 +284,63 @@ test('monitor transfers', async t => {
   await simulateIncomingTransfer();
   t.is(upcallCount, 2, 'no more events after app is revoked');
 });
+
+test('send', async t => {
```

Comment:
please give this a test for sending an ERTP amount, with the expected error message like you did in for the other kit https://github.com/Agoric/agoric-sdk/pull/9870/files#diff-fd9490a6a205a044f2ea98ec20475ad9b5c4ceefd262142af0d76818fc0d216aR60